### PR TITLE
Updated alt-three/testbench

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
         "roumen/feed": "^2.10"
     },
     "require-dev": {
-        "alt-three/testbench": "^1.1",
+        "alt-three/testbench": "^1.3",
```
Comment (GrahamCampbell) [inline composer.json]: No need to bump this. Cachet doesn't require 1.3, only 1.1 or greater.

File: composer.lock [hunk 2]
```diff
@@ -2082,16 +2082,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```
Comment (GrahamCampbell) [inline composer.lock]: Your composer cache is still wrong.

**Comment by GrahamCampbell** [comment]
Please both delete the vendor folder, the composer cache, and the lock file. You're missing one of those steps because composer is still resolving the wrong versions.

File: composer.lock (hunk 2 above)
Comment (jbrooksuk) [inline composer.lock]: I'm telling you, I've removed `vendor`, `composer.lock`, ran `composer clear-cache`. Numerous times.

**Comment by jbrooksuk** [comment]
I've done everything.

**Pull request closed without merging**